websites/U/UpBeatRadio/dist/metadata.json

```diff
@@ -3,13 +3,19 @@
     "name": "Bas950",
     "id": "241278257335500811"
   },
+  "contributors": [
+    {
+      "name": "Marino",
+      "id": "598953747963707409"
+    }
+  ],
```

Comment:
not enough has changed to be able to add yourself as contributor

Reply:
ririxi said so, but ok

Reply:
oof my bad